src/plugins/data_table/data_table.js

```diff
@@ -243,6 +243,47 @@ var LocalDataTable = (function() {
       });
     },
 
+    _renderGrandTotalsRow: function() {
+      var hasGrandTotalsCell = false;
+      // if dataTable is available AND we have a totals row
+      if (this.dataTable && this.totalsRow) {
+        // If we don't have a footer rendered, render it
+        if (this.dataTable.find("tfoot").length < 1) {
+          this.dataTable.append("<tfoot><tr></tr></tfoot>");
+        }
+        // Iterate over the current columns config
+        var grandTotalsRowCellNodes = _.map(this.columnsConfig(), function(col) {
+          // If a column is visible, render its total
+          if (this.columnVisibility(col.id) || col.bulk) {
+            /***
+              TODO: Add some sort of warning if the user has a totals property BUT
+              doesn't have a isNontotalsColumn predicate test.
+            ***/
+            if (this.isNontotalsColumn && this.isNontotalsColumn(col)) {
+              // If column is a non totals column, draw "Grand totals" on the first one and the rest are empty
+              if (hasGrandTotalsCell) {
+                return $("<td></td>");
+              } else {
+                hasGrandTotalsCell = true;
+                return $("<td>Grand totals</td>");
+              }
+            } else { // If the column is NOT a nonTotalsRow column
```

Comment:
please move this trailing comment to new line inside the block.. or actually isn't this line of comment needed? seems pretty obvious by the `if` check above..